From f66963b2a94dda55b653271e57702054accee07d Mon Sep 17 00:00:00 2001
Message-ID: <f66963b2a94dda55b653271e57702054accee07d.1729453324.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1729453324.git.sharlatanus@gmail.com>
References: <cover.1729453324.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Mon, 7 Oct 2024 20:22:47 +0100
Subject: [PATCH 08/42] gnu: python-astropy-iers-data: Update to
 0.2024.10.14.0.32.55.

* gnu/packages/astronomy.scm (python-astropy-iers-data): Update to 0.2024.10.14.0.32.55.
[arguments] <#:tests?>: Adjust comments to reflect current
status. <#:phases>: Remove 'set-env-version' phase.
[native-inputs]: Add python-setuptools and python-wheel; remove
python-pytest and python-pytest-remote-data.

Change-Id: Ia22b52dc41405d73ed313e980f798ae725f47703
---
 gnu/packages/astronomy.scm | 23 ++++++-----------------
 1 file changed, 6 insertions(+), 17 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 43cd5c9ad3..7c83acb185 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -2876,32 +2876,21 @@ (define-public python-astropy-healpix
 (define-public python-astropy-iers-data
   (package
     (name "python-astropy-iers-data")
-    ;; In case of changing the source method git-fetch, consider to check the
-    ;; tag as it's not following the PyPI version, see
-    ;; <https://github.com/astropy/astropy-iers-data/issues/17>.
-    (version "0.2024.9.16.0.32.21")
+    (version "0.2024.10.14.0.32.55")
     (source
      (origin
        (method url-fetch)
        (uri (pypi-uri "astropy_iers_data" version))
        (sha256
-        (base32 "0i63yxw4xfgv1dwaq89xd34xlsnx0n5njcm4adln2gk2ia3gxxig"))))
+        (base32 "1849jbmz1x6x783w9k08a9i3g4f722lqwlnszh4dyr50dyrfbrzg"))))
     (build-system pyproject-build-system)
     (arguments
-     (list
-      ;; Dependencies cycle wit python-astropy, see
-      ;; <https://github.com/astropy/astropy-iers-data/issues/21>.
-      #:tests? #f
-      #:phases
-      #~(modify-phases %standard-phases
-          (add-before 'build 'set-env-version
-            (lambda _
-              (setenv "SETUPTOOLS_SCM_PRETEND_VERSION" #$version))))))
+     (list #:tests? #f)) ; no tests
     (native-inputs
      (list python-hypothesis
-           python-pytest
-           python-pytest-remotedata
-           python-setuptools-scm))
+           python-setuptools
+           python-setuptools-scm
+           python-wheel))
     (home-page "https://docs.astropy.org/en/latest/utils/iers.html")
     (synopsis "IERS Earth Rotation and Leap Second tables for the astropy core package")
     (description
-- 
2.46.0

